prestashop1.7/views/templates/admin/tawkto/helpers/view/view.tpl

```diff
@@ -15,108 +15,45 @@
  * @copyright Copyright (c) 2014-2021 tawk.to
  * @license   http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
  *}
-
 {include file="toolbar.tpl" toolbar_btn=$toolbar_btn toolbar_scroll=$toolbar_scroll title=$title}
 
-{if !$same_user}
-<div id="widget_already_set" style="width: 50%; float: left; color: #3c763d; border-color: #d6e9c6; font-weight: bold;" class="alert alert-warning">Widget set by other user</div>
-{/if}
-
-<iframe
```

Comment:
This iframe was moved to [line 44](https://github.com/tawk/tawk-prestashop/pull/17/files#diff-c02860703bdfe1339eaa7472ceb72f0a6cec22f9302c1f1ba3042a8b81a9b920R44)